src/plugins/controls/public/react_controls/control_group/components/control_panel.test.tsx

```diff
@@ -50,7 +52,7 @@ describe('render', () => {
       await waitFor(() => {
         const controlFrame = controlPanel.getByTestId('control-frame');
         expect(controlFrame.getAttribute('class')).toContain('controlFrameWrapper--medium');
-        expect(controlFrame.getAttribute('class')).toContain('controlFrameWrapper--grow');
+        expect(controlFrame.getAttribute('class')).toContain('euiFlexItem-grow');
```

Comment:
We were never actually using the `controlFrameWrapper--grow` class so I removed it - a `euiFlexItem-grow` class check accomplishes the same thing, though.